requirements-jax-cuda.txt

```diff
@@ -0,0 +1,14 @@
+# Tensorflow (always required for preprocessing).
+tensorflow==2.14.0
+tensorflow-text==2.14.0.rc0
+
+# Torch cpu-only version to assist with testing.
+--extra-index-url https://download.pytorch.org/whl/cpu
```

Comment:
This line looks like a dangling CLI flag, is this right?

Reply:
Actually is. This is the strange way you provide extra urls on requirements files. We need this because jax and torch host a lot of pips outside PyPI